tutorial/tutorial-template.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+{Before using this template, read the accompanying [tutorial template guide](tutorial-template-guide)}.
+
+## Overview
+
+In this tutorial, you'll learn how to {top learning objective} using {product}. This tutorial is intended for {audience}. It assumes you have basic knowledge of:
+
+* Concept 1
+* Concept 2
+* Concept 3...
+
+By the end of this tutorial, you'll be able to:
+
+* Learning objective 1
+* Learning objective 2
+* Learning objective 3...
+
+## Background
+
+{This section is optional. Feel free to use some of the text below to help you get started.}
+
+* {product} is a {product type} that you can use to {common use case}... 
+* {product} provides many of the same features as {competitors}, but with {feature}, you can...
+* Using {feature} enables you to {pain point}...
+
+## Before you begin 
+
+{Use this section to tell users about any steps they need to complete before they start the tutorial.}
+
+Before you start the tutorial, complete these steps:
+
+* Prerequisite 1
+* Prerequisite 2
+* Prerequisite 3...
+
+## {Task name}
+
+To get started, {the first thing your user should do}.
+
+1. {Write the step here. Use a verb to start.}
+
+    {Explanatory text}
+
+    {Optional: Code sample or screenshot that helps your users complete this step}
+
+    {Optional: Result}
+
+2. {Write the step here. Use a verb to start.}
+   
+   a. {Substep 1}
+
+   b. {Substep 1} 
+
+
+
+## Summary
+
+{Use this section to summarize what the user learned in the tutorial.}
+
+In this tutorial, you learned how to:
+
+* Summary point 1
+* Summary point 2
+* Summary point 3..
+
+## Next steps
+
+{Use this section to share links to related tutorials, videos, or other documentation}.
+
+Consider completing some other common tasks using {feature}:
+
+* Task 1
+* Task 2
+* Task 3...
```

Comment:
**Praise:** I really love how clean and bare-bones your latest version of the tutorial is. It makes it a lot more useable. Great work!

Reply:
+1 Great work